_posts/2017-11-07-haskell-learning-group-our-progress-and-updates.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Haskell Learning Group: Our Progress and Updates"
+date: 2017-11-07 00:03:30 +0200
+categories: haskell how-we-learn
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+show_avatar : true
+read_time : 7
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-17-first-haskell-meeting/haskell-poster.jpg
+---
+
+This post is a quick update and the latest news about our Haskell learning group. Even though there were no blog posts on this topic recently, we are still having weekly meetings and mostly stick to the plan. That of course implies preparation time, spend between our sessions. There is no rush in our learning process and we take time to discuss interesting topics a bit longer, especially when all participants are involved. That being said, we might have a plan to go through one chapter per session but it also fine if it is less than that.
+
+As expected the difficulty is slowly increasing as we go along. We are spending more time reading chapters. Sometimes the same material has to be read multiple times in order to fully understand what is it about.
```

Comment:
what is it about -> what it is about.

Actually you maybe want to write "to fully understand it". Because i am pretty sure you understand the *context* (which is what it is about) but maybe not always all details.

Reply:
"what it is about" 👍